### PR TITLE
Plasma Glass Cargo

### PR DESCRIPTION
**Pull request by SethLafuente**
## About the PR 
Added plasma glass into cargo and changed the plasma glass recipe into 2 plasma glass instead of 2 plasma but nobody cares about that players don't even craft plasma glass 
**Screenshots**

image

**Changelog**

:cl: Seth
- add: Added plasma glass crates into cargo

### CODE REVIEW

File: Resources/Prototypes/Entities/Objects/Materials/Sheets/other.yml [hunk 1]
```diff
@@ -92,6 +92,15 @@
...
       - plasma_2
       - plasma_3
+  - type: SolutionContainer
+    caps: None
+    contents:
+      reagents:
+      - ReagentId: Plasma
+        Quantity: 10
+  - type: Tag
+    tags:
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Objects/Materials/Sheets/other.yml:102]: You contaminated your pr..... for a 3rd time.

Comment (SethLafuente) [reply Resources/Prototypes/Entities/Objects/Materials/Sheets/other.yml]: dammit

**Comment by SweptWasTaken** [comment]
You know the drill.

**Comment by SethLafuente** [comment]
> You know the drill.

I don't think I do in all honesty

**Pull request closed without merging**